dashboard/test/ui/features/star_labs/applab/scenarios.feature

```diff
@@ -49,36 +49,39 @@ Feature: App Lab Scenarios
     And I wait until element "#divApplab > .screen > div#text_area1" is visible
     Then element "div#text_area1" has html "Line 1<div>Line 2</div><div><br></div><div>Line3</div>"
 
-  Scenario: Change event works in text input and text area
+  Scenario: Change event works in text input
     Given I switch to design mode
     And I drag a TEXT_INPUT into the app
-    And I drag a TEXT_AREA into the app
     And I switch to code mode
     And I ensure droplet is in text mode
     And I append text to droplet "onEvent('text_input1', 'change', function(event) {\n"
     And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_input1'));\n"
-    And I append text to droplet "});\n\n"
-    And I append text to droplet "onEvent('text_area1', 'change', function(event) {\n"
-    And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_area1'));\n"
     And I append text to droplet "});"
 
     # in text input, blur produces a change event
     When I press "runButton"
-    And I wait until element "#divApplab > .screen > div#text_area1" is visible
-    And I press keys "123" for element "#text_input1"
+    And I wait until element ".screen > input" is visible
+    And I press keys "123" for element ".screen > input"
     And I blur selector "#text_input1"
     Then element "#debug-output" has escaped text "\"text_input1: 123\""
 
     # in a text input, enter produces a change event but then blur does not
-    When I press keys "456\n" for element "#text_input1"
+    When I press keys "456\n" for element ".screen > input"
     Then element "#debug-output" has escaped text "\"text_input1: 123\"\"text_input1: 123456\""
     And I blur selector "#text_input1"
     Then element "#debug-output" has escaped text "\"text_input1: 123\"\"text_input1: 123456\""
 
+Scenario: Change event works in text area
+    Given I switch to design mode
+    And I drag a TEXT_AREA into the app
+    And I switch to code mode
+    And I ensure droplet is in text mode
+    And I append text to droplet "onEvent('text_area1', 'change', function(event) {\n"
+    And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_area1'));\n"
+    And I append text to droplet "});"
+
     # in a text area, blur produces a change event. sending keystrokes (especially 'enter')
     # to a contentetiable div was too hard to test here due to browser differences.
-    When I press "resetButton"
-    And I wait until element "#runButton" is visible
     And I press "runButton"
```

Comment:
Does it matter if this is a "when I press 'runButton'" vs. an "and I press 'runButton'"? I wonder if there are conventions about when you should us "when I ___" vs. "and I ___."

Reply:
According to [this reference](https://cucumber.io/docs/gherkin/reference/) that @breville shared with me since I asked a similar question in [this PR](https://github.com/code-dot-org/code-dot-org/pull/51266#discussion_r1166916993), the key words are interchangeable. However, there are conventions such as the recommendation to have only a single `When` per Scenario. 
I agree that `When I press "runButton"` is more appropriate and matches the pattern in the prior Scenario.
Thanks!

Reply:
This convo leads me to relook at the 'Change event works in text input' scenario. I have 2 'When' key words but it looks like I could actually have 3!

When I press 'runButton' ... then
When I press keys '456\n' ... then
When I blur selector ... then

But the recommendation is to have 1 'When' per scenario. What are your thoughts on this?

Reply:
I think for now I'll stick with the convention of 1 'When' per scenario and use 'And' for the latter steps since it's written that you can use 'And' if 'you have successive Given’s, When’s, or Then’s.'

Reply:
This makes sense to me!